liveblog/liveblog
# [LBSD-2828] - Short disappearing of scheduled posts at the top

### Purpose

This PR fixes a bug where the scheduled post would disappear then reappear on refresh if at the top of the timeline. This was caused when the `lastUpdated` on the embed, which is used to query against the `content_updated_date`, was set using the `_updated` field that was changed in this [PR](https://github.com/liveblog/liveblog/pull/1393) to provide better consistency between normal and scheduled posts. 

### What has changed
- Changed the `last_created_post` and `last_updated_post` to use `content_updated_date` for more consistency with the embed queries.

### Steps to test
1. Open the Editor.
2. Publish normal and schedule posts at different intervals. Make sure the top most post is a scheduled post. 
3. Observe on the live preview that the post shows without disappearing and reappearing.


Resolves: [LBSD-2828](https://sofab.atlassian.net/browse/LBSD-2828)

[LBSD-2828]: https://sofab.atlassian.net/browse/LBSD-2828?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
